tools/src/h5repack/h5repack_main.c

```diff
@@ -771,7 +773,10 @@ parse_command_line(int argc, const char *const *argv, pack_opt_t *options)
                 break;
 
             case 'E':
-                enable_error_stack = 1;
+                if (H5_optarg != NULL)
+                    enable_error_stack = HDatoi(H5_optarg);
```

Comment:
I'm not going to die on that hill today, but strtol is generally a better function than atoi as the former handles errors in a defined way.